tests/phpunit/tests/rest-api/wpRestMenuLocationsController.php

```diff
@@ -120,6 +120,41 @@ public function test_get_item() {
 		$this->assertSame( $menu, $data['name'] );
 	}
 
+	/**
+	 * @ticket 54304
+	 * @covers ::get_items
+	 */
+	public function test_get_items_filter() {
+		$menus = array( 'primary', 'secondary' );
+		$this->register_nav_menu_locations( array( 'primary', 'secondary' ) );
+		add_filter( 'rest_menu_read_access', '__return_true' );
+		$request  = new WP_REST_Request( 'GET', '/wp/v2/menu-locations' );
+		$response = rest_get_server()->dispatch( $request );
+		$data     = $response->get_data();
+		$data     = array_values( $data );
+		$this->assertCount( 2, $data );
+		$names        = wp_list_pluck( $data, 'name' );
+		$descriptions = wp_list_pluck( $data, 'description' );
+		$this->assertSame( $menus, $names );
```

Comment:
Nitpick, could we add some linebreaks after the assertions in test_get_items_filter? This becomes hard to read with all the alignment changes, to me at least. Test readability obviously isn't top priority but it'd be a small improvement in my opinion.